script/tool/lib/src/version_check_command.dart

```diff
@@ -442,11 +442,10 @@ ${indentation}HTTP response: ${pubVersionFinderResponse.httpResponse.body}
     }
 
     if (fromPubspec != fromChangeLog) {
-      printError('''
-${indentation}Versions in CHANGELOG.md and pubspec.yaml do not match.
-${indentation}The version in pubspec.yaml is $fromPubspec.
-${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.
-''');
+      printError(
+          '${indentation}Versions in CHANGELOG.md and pubspec.yaml do not match.\n '
+          '${indentation}The version in pubspec.yaml is $fromPubspec.\n '
+          '${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.');
```

Comment:
Isn't this one the same resulting layout as the ```'''``` version? I generally find it easier to reason about how a multi-line string constant will print since it's more WYSIWYG.